deps/script.jl

```diff
@@ -0,0 +1,3 @@
+# Used by BuildExecutable.jl to build a self-contained executable
```

Comment:
The new `deps/usr/bin/playground` yes. The one used by BuildExecutable.jl won't work on its own.